### ISSUE TITLE
Enabling site to site vpn connection before enabling VPN Gateway doesn't throw an error message

### ISSUE THREAD

**Issue by kiranchavala** [apache/cloudstack#7250]
##### ISSUE TYPE

 * Bug report

##### COMPONENT NAME

~~~
Componet:UI
Componet:api
~~~

##### CLOUDSTACK VERSION

~~~
4.18 nightly
4.17.2

~~~

##### SUMMARY

Enabling site to site vpn connection before enabling VPN Gateway doesn't through an error message

Steps to reproduce the issue 

~~~
1. Create a new vpc

2. Create Vpn customer gatweway

3. Click > VPC > VPN connection > Create site to site vpn connection ()

4. The ui job goes on , on  Refreshing the page , message says success

6. Ideally The UI has to throw a popup to create VPN Gateway first before creating VPN connection
~~~

[Screencast from 17-02-23 01:00:11 PM IST.webm](https://user-images.githubusercontent.com/1401014/219579630-7c269904-3632-450f-b7c6-9ae67ed6e04f.webm)

**Comment by davidjumani** [comment]
This is more of an enhancement to warn the user rather than a bug

**Comment by DaanHoogland** [comment]
fixed in #9195
